LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2014 Segment Inc. friends@segment.com
```

Comment:
Why 2014?

Reply:
https://github.com/segmentio/analytics-php#license

Year indicates the first release of the software.

Reply:
IANAL, but I think this should be the latest year rather than the year of first release? I'll merge this for now, but is there any resources advocating for one way or the other?

Reply:
IANAL as well. 

https://www.copyrightlaws.com/copyright-notice-year/

Reply:
Thanks!